apps/style/common.scss

```diff
@@ -1586,7 +1586,24 @@ a.WireframeButtons_active, div.WireframeButtons_active {
   background-size: 432px 757px;
 }
 
+@media screen and (max-height: 780px) {
```

Comment:
This doesn't seem like a great experience for our 1024x768 users.  I asked Poorva to take a look and see if this matched her expectation.

Reply:
yes, I had sent the screenshot to her, but she may have misunderstood. I've updated it to move the footer closer to the visualization in this mode. Screenshot above updated with a playlab example.